Developer mv

MoniFerVivas opened the pull request:
Cambio en Controlador listar contratos
Aumento de un método en Servicio Contrato

MoniFerVivas committed:
Cambio controlador y servicio Contrato

MoniFerVivas committed:
Merge branch 'developer' into developer_MV

dnroar commented on src/main/java/GrupoF/Proyecto3/Controladores/ControladorContrato.java:
```diff
@@ -75,17 +75,19 @@ public String registrarContrato(HttpSession session, @RequestParam (name = "cate
     @PreAuthorize("hasAnyRole('ROLE_USUARIO')")
-    @PostMapping("/listaContratos")
-    public List listarContratos(HttpSession session, @RequestParam String modo) throws MiExcepcion {
+    @PostMapping("/contratos/listaContratos")
```
@MoniFerVivas  de acuerdo a la teoría impartida en guía de aprendizaje, no es necesario incluir la Raiz RequestMapping nuevamente. Voy a corregirlo, por ti, en vista del tiempo, y se un cambio pequeño.

dnroar committed:
Update ControladorContrato.java

Se corrije POSTMAPPING, no es necesario incluir la Raiz REQUESTMAPPING en el POSTMAPPING.

dnroar reviewed (approved):
Integridad: Riesgo Bajo / CleanCode: Ok.
